project/npda/templates/dashboard.html

```diff
@@ -1,72 +1,75 @@
-    </div>
+<div class="container mx-auto p-4 font-montserrat text-rcpch_dark_blue">
+  <div class="my-6">
+    <h1 class="text-5xl text-center">PDU Dashboard for {{pdu_lead_organisation.name}} ({{pdu_object.pz_code}})</h1>
+    <p class="text-center text-gray-500 mt-2">Calculated at {{ kpi_calculations_object.calculation_datetime }}</p>
+    <p class="text-center">This data has not been validated by the NPDA team.</p>
   </div>
-</section>
-<div class="container-mx-auto my-20 mt-10 flex justify-center">
-  <div class="card bg-base-100 w-96 shadow-xl">
-    <figure>
-      <div id="organisation_cases_map" style="width: 100%; height: 700px"></div>
-    </figure>
-    <div class="card-body">
-      <h2 class="card-title">Patients</h2>
-        <p>Distribution of all patients at {{ session.pz_code }}</p>
+  <div class="grid grid-cols-1 md:grid-cols-3 gap-4">
+    <!-- PDU Information Card -->
+    {% include 'dashboard/pdu_card.html' %}
+
+
+    <!-- Audit Details Card -->
+    {% include 'dashboard/audit_details_card.html' %}
+
+    <!-- Total Eligible Patients Card -->
+    {% include 'dashboard/total_eligible_patients_card.html' %}
+  </div>
+
+  <!-- UNIT REPORT -->
+  <h1 class="text-5xl text-center my-6">Unit Report</h1>
+  <div class="grid grid-rows-3 gap-4">
+    <div class="grid grid-cols-1 md:grid-cols-3 gap-4">
+      <!-- PATIENT CHARACTERISTICS CARD -->
+      {% include 'dashboard/patient_characteristics_card.html' %}
+      <!-- TREATMENT REGIMEN CARD -->
+      {% include 'dashboard/treatment_regimen_card.html' %}
+      <div class="flex flex-col gap-3">
+        <!-- HCL CARD -->
+        {% include 'dashboard/hcl_use_card.html' %}
+        <!-- care at diagnosis CARD -->
+        {% include 'dashboard/care_at_diagnosis_card.html' %}
+      </div>
+    </div>
+    <div class="grid grid-cols-1 md:grid-cols-3 gap-4">
+      <!-- HEALTH CHECKS COMPLETION RATE CARD -->
+      {% include 'dashboard/health_checks_completion_rate_card.html' %}
+      <!-- ADDITIONAL CARE PROCESSES CARD -->
+      {% include 'dashboard/additional_care_processes_card.html' %}
+      <div class="flex flex-col gap-3">
+        <!-- HBa1c CARD -->
+        {% include 'dashboard/hba1c_card.html' %}
+        <!-- ADMISSIONS CARD -->
+        {% include 'dashboard/admissions_card.html' %}
+      </div>
     </div>
+    <div class="grid grid-cols-1 md:grid-cols-4 gap-4">
+      <!-- SEX CARD -->
+      {% include 'dashboard/sex_card.html' %}
+      <!-- ETHNICITY CARD -->
+      {% include 'dashboard/ethnicity_card.html' %}
+      <!-- IMD CARD -->
+      {% include 'dashboard/imd_card.html' %}
+      <!-- IMD MAP CARD -->
+      {% include 'dashboard/imd_map_card.html' with scatterplot_of_cases_for_selected_organisation=charts.scatterplot_of_cases_for_selected_organisation pdu_lead_organisation=pdu_lead_organisation pdu_object=pdu_object %}
+
+    </div>
+
+  <!-- PATIENT LEVEL REPORT -->
+  <h1 class="text-5xl text-center my-6">Patient-Level Report</h1>
```

Comment:
so this is really nice @anchit-chandran but it is in the wrong place. It belongs on the patient detail view below the visit list. We could signpost here that patient level stats are available, and I wonder if we should do something with colour and naming to clarify to users what the different views are. So this would be Dashboard View, the patient list might be Patient List, and then the patient detail might be Patient Visits. Probably the visit form itself is self explanatory and does not need a name